### ISSUE TITLE
Little Problems and Suggestions with the Interface/Documentation/Help

### ISSUE THREAD

**Issue by jounathaen** [mrdokenny/Cookie-AutoDelete#275]
Hey, I use this Add On, since my old Add on "Self-Destructing Cookies"is not available anymore in the new Firefox (Quantum).
But I had some Problems understanding the Program and the Interface, so here is my 5 Cents about this:

- In the README.md it says `Enable “Active Mode” in the popup or settings`. I only find "Auto Clean Enabled/Disabled". I assume this just needs to be updated.

- The Popup of the `Auto Clean Enabled/Disabled` Button is "Toggle to Disable/Enable automatic Cleanup". This does not help at all. I'd rather suggest something like "Automatic Cleanup deletes all Cookies after the Browser is closed"
The same goes for the other Buttons: 
`Clean`: "Run Cleanup Now" But it should be "Deletes all unwanted Cookies NOW, excluding openTabs"
`Clean, include open Tabs` Same as above.

- There is a nice FAQ here on Github: [Documentation/FAQ](https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/wiki/FAQ:-Common-Questions-and-Issues), but I can't find the link in the Pop-Up and in the settings it is hidden in `Cookie Settings` -> *Question Mark Button*. I'd put the on top of the `Welcome` page of the settings.

- The FAQ (see above) is already quite good, but I think for a Newbie, there should be a short explanation of the terms "Clean" and "Greylist/Whitelist". Maybe even a short paragraph about what a "Cookie" is. At the moment some of this information is on the README.md, but this is hard to find. In overall i think the Documentation is a bit to focused on experienced Web users.

Best Regards and thanks for developing Open Source Software!

- OS/version: Linux 
- Browser/version: FF 57 64Bit
- Cookie AutoDelete version: I don't know where to find this information

**Comment by kennethtran93** [comment]
Not sure why this wasn't communicated earlier.  There's been a whole lot of change since your post and some, if not all, of your suggestions may have already been implemented.  Is there anything else that needs to be looked into?

> Hey, I use this Add On, since my old Add on "Self-Destructing Cookies"is not available anymore in the new Firefox (Quantum).
> But I had some Problems understanding the Program and the Interface, so here is my 5 Cents about this:
> 
>     * In the README.md it says `Enable “Active Mode” in the popup or settings`. I only find "Auto Clean Enabled/Disabled". I assume this just needs to be updated.
> 
>     * The Popup of the `Auto Clean Enabled/Disabled` Button is "Toggle to Disable/Enable automatic Cleanup". This does not help at all. I'd rather suggest something like "Automatic Cleanup deletes all Cookies after the Browser is closed"
>       The same goes for the other Buttons:
>       `Clean`: "Run Cleanup Now" But it should be "Deletes all unwanted Cookies NOW, excluding openTabs"
>       `Clean, include open Tabs` Same as above.
> 
>     * There is a nice FAQ here on Github: [Documentation/FAQ](https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/wiki/FAQ:-Common-Questions-and-Issues), but I can't find the link in the Pop-Up and in the settings it is hidden in `Cookie Settings` -> _Question Mark Button_. I'd put the on top of the `Welcome` page of the settings.

There are `Documentation` and `FAQ` links in the About page already.

> 
>     * The FAQ (see above) is already quite good, but I think for a Newbie, there should be a short explanation of the terms "Clean" and "Greylist/Whitelist". Maybe even a short paragraph about what a "Cookie" is. At the moment some of this information is on the README.md, but this is hard to find. In overall i think the Documentation is a bit to focused on experienced Web users.

This is supposed to be more or less geared towards users that have at least intermediate Web browsing terminology.   I believe if one is a basic web user, then they shouldn't really need this as it will affect their normal usage.  Privacy-enhanced users should already understand the web terminology enough as well.

**Comment by jounathaen** [comment]
Hi,
I don't think all of the mentioned concerns are implemented as of 3.4


> ~In the README.md it says Enable “Active Mode” in the popup or settings. I only find "Auto Clean Enabled/Disabled". I assume this just needs to be updated.~

This is resolved :+1: 


>     The Popup of the Auto Clean Enabled/Disabled Button is "Toggle to Disable/Enable automatic Cleanup". This does not help at all. I'd rather suggest something like "Automatic Cleanup deletes all Cookies after the Browser is closed"
>     The same goes for the other Buttons:
>     Clean: "Run Cleanup Now" But it should be "Deletes all unwanted Cookies NOW, excluding openTabs"
>     Clean, include open Tabs Same as above.

I still think the popup texts could be improved. Not that they are useless but "Toggle to disable automatic cleanup" on the "Auto Clean enabled" button is not really a helpful addition.


>  ~There is a nice FAQ here on Github: Documentation/FAQ, but I can't find the link in the Pop-Up and in the settings it is hidden in Cookie Settings -> Question Mark Button. I'd put the on top of the Welcome page of the settings.~

You are right, it is on the about page. I think it is stil a bit hidden, especially if you consider that the information presented on the welcome page is the changelog which I'd consider not as important as the FAQ. But I'd say this concern is resolved.

>>     The FAQ (see above) is already quite good, but I think for a Newbie, there should be a short explanation of the terms "Clean" and "Greylist/Whitelist". Maybe even a short paragraph about what a "Cookie" is. At the moment some of this information is on the README.md, but this is hard to find. In overall i think the Documentation is a bit to focused on experienced Web users.
> This is supposed to be more or less geared towards users that have at least intermediate Web browsing terminology. I believe if one is a basic web user, then they shouldn't really need this as it will affect their normal usage. Privacy-enhanced users should already understand the web terminology enough as well.

This is basically resolved as there is a short paragraph on the README about cookies and the tooltips of the greylist and whitelist buttons are telling a bit what they are doing. This could be extended but somehow the information is there.
 
But nevertheless I strongly disagree with the argument that privacy enhancing is reserved for "advanced users" only. I'd always say that you should aim to make privacy so easy that everybody can understand and use it.

**TLDR:** Tooltips could be improved, FAQ is a bit hidden, the other concerns are resolved

**Comment by kennethtran93** [comment]
> I still think the popup texts could be improved. Not that they are useless but "Toggle to disable automatic cleanup" on the "Auto Clean enabled" button is not really a helpful addition.

The button text shows the current status, and the mouse over text shows the action after clicking the button.  If you have any suggestions then please feel free to put them here.  There's already the appended `(manual mode)` and `(automatic mode)` to that text that additionally helps the user out too.  Notification button also shows the current status, but the mouse over text, however, is the same for both states.

> You are right, it is on the about page. I think it is stil a bit hidden, especially if you consider that the information presented on the welcome page is the changelog which I'd consider not as important as the FAQ. But I'd say this concern is resolved.

I suppose it wouldn't hurt to add a documentation and FAQ links to the Welcome Page.

**Comment by kennethtran93** [comment]
Documentation and FAQ links will be on the Welcome page as of 3.5.0, per PR #798.  Seeing as there's no further suggestions for the tooltips I'll close this as resolved when the pr gets merged.
